src/SFA.DAS.AANHub.Application.UnitTests/Partners/Queries/GetPartnerMemberQueryValidatorTests.cs

```diff
@@ -0,0 +1,25 @@
+﻿using FluentValidation.TestHelper;
+using NUnit.Framework;
+using SFA.DAS.AANHub.Application.Partners.Queries;
+
+namespace SFA.DAS.AANHub.Application.UnitTests.Partners.Queries
+{
+    [TestFixture]
+    public class GetPartnerMemberQueryValidatorTests
+    {
+        [TestCase("username", true)]
+        [TestCase(null, false)]
+        [TestCase("", false)]
```

Comment:
Mirroring feedback once more but I'd add a test for whitespace, just in case
```suggestion
        [TestCase(" ", false)]
```

Reply:
Applied recommendation